test/dbservice/groups_test.exs

```diff
@@ -8,11 +8,28 @@ defmodule Dbservice.GroupsTest do
 
     import Dbservice.GroupsFixtures
 
-    @invalid_attrs %{input_schema: nil, locale: nil, locale_data: nil}
+    @invalid_attrs %{
+      name: nil,
+      type: nil,
+      program_type: nil,
+      program_sub_type: nil,
+      program_mode: nil,
+      program_start_date: nil,
+      program_target_outreach: nil,
+      program_donor: nil,
+      program_state: nil,
+      batch_contact_hours_per_week: nil,
+      group_input_schema: nil,
+      group_locale: nil,
+      group_locale_data: nil
+    }
 
     test "list_group/0 returns all group" do
       group = group_fixture()
-      assert Groups.list_group() == [group]
+      [head | _tail] = Groups.list_group()
+      assert is_list(Groups.list_group()) == is_list([group])
```

Comment:
Also, it looks like we're comparing two boolean values here. I don't think that's a valid check. `boolean == boolean` looks improper.

We only need to check if the output of list group is a valid list. We don't need to do anything with group fixture I think.